packages/manager/apps/pci-private-network/src/hooks/usePageTracking.ts

```diff
@@ -1,7 +1,7 @@
 import { useEffect } from 'react';
```

Comment:
we could probably externalize this hook in a separated module and reuse it in each pci-app to avoid code duplication

Reply:
I'll note that, but maybe in another PR, what do you think ?

Reply:
yes sure, let's do this in a separated task